### DISCUSSION TITLE
Filter on integrations page to display only failed integrations

### DISCUSSION THREAD

**Discussion by dougiteixeira** [home-assistant/frontend#16965] [Configuration]
The new integrations page looks great! But I miss a filter on the integrations page to display only the integrations that are failing in any of the configuration entries.

Please consider adding this feature.

**Comment by Mariusthvdb** [comment]
+1 to that.
also, if all are shown, including the failing ones, it would be nice when these would be grouped atop, and not have the user force scrolling the whole integrations page.... which is a nasty task especially on mobile
